.travis.yml

```diff
@@ -56,6 +56,17 @@ matrix:
         - echo '{"ipv6":true,"fixed-cidr-v6":"2001:db8:1::/64"}' | sudo tee /etc/docker/daemon.json
         - sudo service docker restart
 
+    - rvm: 2.5.1
+      env:
+        - BEAKER_PUPPET_COLLECTION=puppet5
+        - BEAKER_setfile=debian10-64{hostname=debian10-64.example.com}
```

Comment:
There is no puppetserver package available in that combination.

Reply:
Oh yes, automatically ran modulesync and didn't manually check the results. Forgot this one exception.